pkg/sloop/webfiles/sloop_ui.js

```diff
@@ -242,9 +240,14 @@ function bindMouseEvents(svg) {
                 }
             );
             tooltip.html(resourceBarHtml);
+            positionTooltip(tooltipX, tooltipY);
```

Comment:
is this going to cause a flicker (visible for a short time in original location/size, then move/resize based on additional data)?